### PR TITLE
Script for syncing Heroku DBs to local machine

### PR DESCRIPTION
**Pull request by tom-mullen**
Why:
- Getting your DB looking like production is handy!

This change addresses the need by:
- Creating a script to make backing-up, downloading and restoring a
  remote DB a cinch.

### CODE REVIEW

**Comment by tom-mullen** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;30LfuCJo&#x2F;225-dvelp-add-script-for-syncing-remote-db-to-local"> DVELP - Add script for syncing remote DB to local</a>

File: scripts/README.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 A collection of shell scripts that we use at DVELP for our making the everyday more efficient.
 
+# Heroku Sync DB to Local
+
+Get it: [heroku\_sync_db_to_local.sh](https://github.com/DVELP/cookbook/blob/master/scripts/git_heroku_sync_db_to_local.sh)
```
Comment (vfonic) [inline scripts/README.md:7]: There's extra `git_` in the script name

File: scripts/heroku_sync_db_to_local.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/zsh
+# Script for syncing remote DB to local
+
+APP_NAME=$1
```
Comment (vfonic) [inline scripts/heroku_sync_db_to_local.sh:4]: Could we get the app name from the current directory, like heroku does? We could still have $APP_NAME as optional parameter.

File: scripts/heroku_sync_db_to_local.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+DB_NAME=$2
+DUMP_FILE='latest_backup.dump'
+
+if [ -n "$APP_NAME" ] && [ -n "$DB_NAME" ]; then
+
+  # Create new backup
+  echo "Creating backup for $APP_NAME..."
+  heroku pg:backups capture --app $APP_NAME
```
Comment (vfonic) [inline scripts/heroku_sync_db_to_local.sh:12]: This step here is the slowest one. Probably takes more than 50% of the time and I'd say it's not always needed. We could make it optional by passing some parameter like `--no-backup`

Comment (tom-mullen) [inline scripts/heroku_sync_db_to_local.sh:12]: I have added this and the directory name default to the 'to do' section in the read me.

**Pull request merged**